app/assets/stylesheets/application.scss

```diff
@@ -1,8 +1,12 @@
 @import "govuk_publishing_components/all_components";
+
 @import "./components/miller-columns";
 @import "./components/autocomplete";
+
 @import "./modules/unpublish-display-conditions";
 
+@import "./pages/publisher_updates";
```

Comment:
It doesn't look like we have a pattern for putting in stylesheets in a pages directory? Wouldn't convention dictate that this should be in admin/views ?

Reply:
We can move it to `views` but I wanted to move away from the `admin` directory as all the legacy CSS is there. Also now that we have removed all the frontend code, there's no need to have this seperation.

Reply:
We haven't removed the frontend code yet, there's no firm ETA on that so should be cautious on assumptions there. If we create a new convention without coordinating it we just leave another WTF in the codebase.

Reply:
I don't think we should have it under admin as we require the admin.scss (which requires the whole folder) for the legacy scope. It's interesting you raised this point as we should probably rename the admin folder as `admin_legacy` so we know (and anyone in the future) will know that all the legacy CSS sits in there. Once we rename it, it will be clear what is current and what is legacy.

Reply:
Hey, I've just picked up reviewing this today from standup. Are you still working on where these files go? As they're still not in an admin directory?

Reply:
I'm going to leave it where it is and create a separate PR to rename the bootstrap stuff

Reply:
I'd suggest putting it into admin so it's at least not in an inconsistent place before doing fixes in a later PR.

Reply:
Done